### PR TITLE
[scheduler] fix some scheduler dtype error

### PR DESCRIPTION
**Pull request by furry-potato-maker**
when use nvidia gpu, the dtype of timesteps is float64 and the dtype of timesteps_interpol is float32, but they will be pushed into same torch.stack. This bug will make some optimization failed. For example, when I use oneflow to mock torch, torch.stack will push timesteps_interpol and timesteps which has different dtype, and then oneflow will clash.

### CODE REVIEW

**Comment by patrickvonplaten** [comment]
Test failures are unrelated - good to go for me
